content/rs/administering/_index.md

```diff
@@ -5,10 +5,24 @@ weight: 50
 alwaysopen: false
 categories: ["RS"]
 ---
-This section covers everything you need to know to get up and running
-with RS, included installing, configuring a cluster, maintaining a
-cluster, creating databases, etc.
+This section covers everything you need to know to get up and running with RS.
 
-Topics include:
+{{% note %}}
+In addition to using the RS web UI to manage the cluster,
+you can also access cluster operations through the cluster REST API.
+
+To access the cluster REST API documentation, either:
+
+- Open the HTML files in the RS software package:
+    1. Download the RS software package.
+    1. Unarchive the package.
+    1. Unarchive the rlec_docs.tar.gz package.
+    1. Open the `rlec_docs\html\index.html` file in a web browser.
+- Open the HTML files in an installed RS node:
+    1. Connect to the terminal of an RS node.
+    1. Change directory to `/opt`.
+    1. Download and unarchive the rlec_rest_api.tar.gz package.
```

Comment:
Download? is should already be there

Reply:
I meant to move it to another machine to view it in a browser. I made it more clear.